config/locales/alias.en.yml

```diff
@@ -0,0 +1,10 @@
+en:
+  booth:
+    one: 'stand'
+    other: 'stands'
+  Booth:
```

Comment:
this is the same capitalized. I would use in the ruby code [capitalize](https://ruby-doc.org/core-2.6/String.html#method-i-capitalize) instead, so that when modifying this file there are no repeated entries.

Reply:
@Ana06 makes sense :sweat_smile:, I will use capitalize.